### ISSUE TITLE
Code editor scrolls back when moving cursor to end of line with keyboard

### ISSUE THREAD

**Issue by maltenuhn** [concrete-utopia/utopia#1024]
**Describe the bug**
When encountering a line of code that's longer than the available window, and using CMD+Right to jump to the end of the line, the editor jumps, but then scrolls back 

**To Reproduce**
1. Type in a line of code that stretches beyond the visible right edge of the code editor, eg
```
<div
        style={{
          width: 270,
          height: 215,
          background: 'hsl(0,0%,97%)',
          boxShadow: 'rgba(0, 0, 0, 0.1) 0px 0px 0px 1px, rgba(0, 0, 0, 0.1) 0px 4px 11px',
          backgroundColor: 'rgb(247, 247, 247, 1)',
        }}
      >Hello</div>
```
2. (if required, make the code editor pane smaller)
3. Position the cursor in the line containing `box-shadow`
4. Press CMD+Right (or repeatedly, Right)
5. Cursor moves correctly, editor scrolls, and scrolls back

**Expected behaviour**
Cursor moves, window scrolls and stays scrolled

**Screenshots**
![Large GIF (692x468)](https://user-images.githubusercontent.com/2945037/113027353-dda02200-9181-11eb-8924-1eaa5928c50b.gif)

**Comment by Rheeseyb** [comment]
I'm pretty sure this is related to #1025 and is caused by the syncing of selection between Utopia and VSCode

**Comment by Rheeseyb** [comment]
Closed by #1100
